Source/AliveLibCommon/AnimResources.hpp

```diff
@@ -162,19 +162,61 @@ enum class AnimId
     Glukkon_Phleg_Unknown_K,
     Glukkon_Phleg_Walk,
 
+    Greeter_Chase,
+    Greeter_Falling,
+    Greeter_Hit,
     Greeter_Idle,
-    Greeter_Turn_Around,
-    Greeter_Shot,
     Greeter_Speak,
-    Greeter_Chase,
+    Greeter_Turn_Around,
+
+    Slog_Bark,
+    Slog_Bark_Short_A,
+    Slog_Bark_Short_B,
+    Slog_BROKEN_J,
+    Slog_Die,
+    Slog_Eat,
+    Slog_Falling,
+    Slog_Idle_A,
+    Slog_Idle_B,
+    Slog_Jump_Up,
+    Slog_Lunge,
+    Slog_Run,
+    Slog_Run_Stop,
+    Slog_Run_Turn_Around,
+    Slog_Scratch,
+    Slog_Sleep_A,
+    Slog_Sleep_B,
+    Slog_Sleep_C,
+    Slog_Turn_Around,
+    Slog_Unknown_A,
+    Slog_Unknown_B,
+    Slog_Unknown_C,
+    Slog_Unknown_D,
+    Slog_Unknown_E,
+    Slog_Walk,
+
+    Slurg_Burst,
+    Slurg_Move,
+    Slurg_Turn_Around,
 
     Grenade,
     Lift_Rope,
+    Timed_Mine_Armed,
+    Timed_Mine_Idle,
     Security_Door_Idle,
 	Security_Door_Speak,
     Security_Orb,
-    Slap_Lock_Idle,
+    Slap_Lock_Idle_A,
+	Slap_Lock_Idle_B,
```

Comment:
a strange mix of spaces/tabs in most places, needs to be spaces cause that is what everything else is using